tests/test_aampdist_snippets.py

```diff
@@ -7,8 +7,8 @@
 
 
 test_data = [np.random.uniform(-1000, 1000, [64]).astype(np.float64)]
-s = [6, 7, 8]
-percentage = [0.7, 0.8, 0.9]
+s = [3, 6, 7, 8]
```

Comment:
Why are these added in `test_aampdist_snippets.py` rather than `test_snippets.py`?

Reply:
I wanted to make sure we do not have the problem in `test_aampdist_snippets.py`. I also added them in `test_snippets.py`, but then I got failure, so I decided to create two new test functions for them. So, in the new test function, I am testing the function with `s` that has low value. Same goes for `percentage`. However, now that the test suite is passing, I think I should better try putting the value `3` back into the list and see how it geos.

Reply:
@seanlaw 
So, I tested `snippets.py` after adding `3` to list `s`, i.e. `s=[3, 6, 7, 8]`. I did the same for percentege: `percentage = [0.3, ...]`
And, I got failure. I think it is related to precision. 

Do you want me to push it so you can see  it here?

Reply:
Sure